aea/cli/test.py

```diff
@@ -494,7 +508,7 @@ def test_package_by_path(
             ),
             *pytest_arguments,
         ]
-        exit_code = pytest.main(runtime_args)
+        exit_code = _call_pytest(runtime_args)
```

Comment:
That's an out of scope change. Why has it been done?

Reply:
@DavidMinarsch 
there was an interesting issue:
`aea test by-path` coverage differs from `pytest ./` call.

 i checked the problem: running pytest from python has effect on coverage. so now pytest executed in a subprocess in a clean python process and match pytest coverage.

Reply:
moved out to https://github.com/valory-xyz/open-aea/pull/529